index.html

```diff
@@ -15,7 +15,7 @@
             <img href src="./image/logobuger.JPG">
      </nav>
 </header>
```

Comment:
- [ ] **[Optional]** Kindly try to fix the text wrapping the logo to replicate that of figma design :joy: 

**Your design** 
![image](https://user-images.githubusercontent.com/102757126/205449391-8c1a3537-561f-46e8-8e26-1efccd52119e.png)